metrics/web/lib/common/presentation/injector/widget/injection_container.dart

```diff
@@ -61,13 +84,43 @@ class _InjectionContainerState extends State<InjectionContainer> {
             _signOutUseCase,
           ),
         ),
-        ChangeNotifierProvider<ProjectMetricsNotifier>(
-          create: (_) => ProjectMetricsNotifier(
-            _receiveProjectUpdates,
-            _receiveProjectMetricsUpdates,
+        ChangeNotifierProvider<ThemeNotifier>(
+          create: (_) => ThemeNotifier(),
+        ),
+        ChangeNotifierProxyProvider<AuthNotifier, ProjectsNotifier>(
+          create: (_) => ProjectsNotifier(_receiveProjectUpdates),
+          update: (_, authNotifier, projectsNotifier) {
+            return projectsNotifier
+              ..updateProjectsSubscription(
+                isLoggedIn: authNotifier.isLoggedIn,
+              );
+          },
+        ),
+        ChangeNotifierProxyProvider<ProjectsNotifier, ProjectMetricsNotifier>(
+          create: (_) => ProjectMetricsNotifier(_receiveProjectMetricsUpdates),
+          update: (_, projectsNotifier, projectMetricsNotifier) {
+            return projectMetricsNotifier
+              ..updateProjects(
+                projectsNotifier.projects,
+                projectsNotifier.errorMessage,
```

Comment:
Can we discuss this part? I want to check that I understand errorMessage here and why do we pass it to the notifier.